convert/commands/base.py

```diff
@@ -79,16 +79,18 @@ def get_user_input(self, conversion_data):
         # display warning
         prmsg('warning')
 
+        # multiple files flow
         if self.options['--multiple']:
+            # do not show confirmation message if the option is enabled
+            if not self.options['--no-confirm']:
```

Comment:
This bit is copied in other branch of this conditional. Have you thought about pulling it out into a separate function?

Reply:
Hmmm, I didn't, it would be kind of the same though...

```
def no_confirm():
    return self.options['--no-confirm']
```

Then in the code you quoted:
```
if not no_confirm():
```

Can you give an example of how you would do it?
I am confused because it is not a piece of code, it is a one liner.

Reply:
Sorry, I did not mean this one line but rather block of code.
```            # do not show confirmation message if the option is enabled
            if not self.options['--no-confirm']:
                if not self.confirm_multiple(source_extension,
                                             source_folder,
                                             conversion_data['extension'],
                                             destination):
                    sys.exit(2)

                # clear screen
                clear()
```

The difference between is a function call or either `confirm_multiple` or `confirm_single`.

```
def confirmator(confirm_func, *args):
            if not self.options['--no-confirm']:
                if not confirm_func(*args):
                    sys.exit(2)

                # clear screen
                clear()
```

This is a really raw pseudo solution but maybe it will illustrate what I have on mind.

Reply:
Oh, yes, I see it.

Thanks, I will give it a go on this PR, cheers!